packages/devextreme-scss/scss/widgets/material/chat/_colors.scss

```diff
@@ -0,0 +1,27 @@
+@use "sass:color";
+@use "../sizes" as *;
+@use "../colors" as *;
+
+// adduse
+
+$chat-background-color: $base-bg !default;
+$chat-messagebox-border-color: $base-border-color !default;
+$chat-avatar-background-color: #878787 !default;
+$chat-bubble-background-color-primary: null !default;
+$chat-bubble-background-color-secondary: null !default;
+$chat-information-color: null !default;
+$chat-messagebox-button-color: null !default;
+
+@if $mode == "light" {
+  $chat-bubble-background-color-primary: #03A9F414 !default;
```

Comment:
All colors should be calculated from base variables to support ThemeBuilder

Reply:
The mathematical calculations of the colours look strange to me. In my opinion, all colours should be set separately without mathematical calculations on a basic level.

Reply:
[b5bbab3e3d2663f50b15f1c50726ddb07f6b6094]